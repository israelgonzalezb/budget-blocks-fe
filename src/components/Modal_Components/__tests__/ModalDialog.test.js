
// Auto-generated do not edit


/* eslint-disable import/no-extraneous-dependencies */
/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import ModalDialog from '../ModalDialog';


describe('ModalDialog test', () => {
  it('ModalDialog should match snapshot', () => {
    const component = renderer.create(<ModalDialog
      handleClose={undefined} modalTitle={undefined} maxWidth={undefined} ariaLabelledBy={undefined} handleSubmit={undefined} backContinue={undefined} dialogActions={undefined} submitEdit={undefined} open={undefined} />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
