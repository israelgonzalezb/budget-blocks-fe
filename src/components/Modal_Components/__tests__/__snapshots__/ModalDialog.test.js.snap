// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModalDialog test ModalDialog should match snapshot 1`] = `null`;
